docker/web/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ "$(id -u)" == '0' ]; then
+  # Set PostgreSQL password file from secrets.
+  pgpass=/run/secrets/pgpass
+  if [ -f $pgpass ]; then
+    cp $pgpass /.pgpass
+    chmod 0600 /.pgpass
+    chown codecompass:codecompass /.pgpass
+    export PGPASSFILE=/.pgpass
+  fi
```

Comment:
Why do we have this PostgreSQL related part if SQLite is used in this image? If we intend to support both database engines then we should document better how to do it.